core/new-gui/src/app/workspace/service/workflow-graph/model/mock-workflow-data.ts

```diff
@@ -39,6 +40,7 @@ export const mockSentimentPredicate: OperatorPredicate = {
   operatorType: 'NlpSentiment',
   operatorProperties: {
   },
+  operatorGroupName: 'Analytics',
```

Comment:
revert all the changes related to `operatorGroupName`, if you need to change an operator's group in the mock data, change them in `mock-operator-metadata.data.ts`